### PR TITLE
First pass at lesson 1

### PR DESCRIPTION
**Pull request by escht**
Homework 1: Read name, surname, age, weight (in kg) and height (in cm) from console

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,36 @@ class Program
...
+
+            Console.WriteLine(firstName[0] + " " + lastName[0] + " is " + age[0] + " years old, her weight is " + weight[0] + "kg and her height is " + height[0] + " cm.");
+            Console.WriteLine(firstName[1] + " " + lastName[1] + " is " + age[1] + " years old, his weight is " + weight[1] + "kg and his height is " + height[1] + " cm.");
+            Console.WriteLine(firstName[2] + " " + lastName[2] + " is " + age[2] + " years old, his weight is " + weight[2] + "kg and his height is " + height[2] + " cm.");
+
+            // Formula: weight (kg) / [height (m)]2
+            // Calculation: [weight(kg) / height(cm) / height(cm)] x 10,000
+            // cm to m --> divide by 100
+
+            var toMeters0 = (height[0] / 100);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: No need brackets there.

Comment (Almantask) [reply Src/BootCamp.Chapter/Program.cs]: And in multiple other places

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,36 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+           string[] firstName = { "Josie", "Daniel", "Liam" };
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: The heights, weights and names are hardcoded. Could you make it more like it was required- by reading user input?

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4842747&bid=16077284)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,38 @@ class Program
...
+            Console.WriteLine(weight);
+
+            Console.WriteLine(firstName + " " + lastName + " is " + age + " years old, their weight is " + weight + "kg, their height is " + height + " cm and their BMI is ");
+
+            // BMI formula: weight (kg) / [height (m)]2
+            // Calculation: [weight(kg) / height(cm) / height(cm)] x 10,000
+            // cm to m --> divide by 100
+
+            // my code.. this is where I am having trouble
+            // var toMeters = height / 100;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4842747)

Comment (escht) [reply Src/BootCamp.Chapter/Program.cs]: i kept that there to check if i was on the right path, but I guess that's not good practice?

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (escht) [reply Src/BootCamp.Chapter/Program.cs]: gotcha, all removed.

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -6,7 +6,28 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
```
Comment (diverges) [inline Src/BootCamp.Chapter/Program.cs]: Remember you have to calculate BMI for another person :)

Comment (escht) [reply Src/BootCamp.Chapter/Program.cs]: okay, I added this! but I am having a hard time understanding how to resolve the division issue.

File: Src/BootCamp.Chapter/Program.cs [hunk 5]
```diff
@@ -6,7 +6,52 @@ class Program
...
+            string firstName2 = Console.ReadLine();
+
+            Console.Write("Last name: ");
+            string lastName2 = Console.ReadLine();
+
+            Console.Write("Age: ");
+            var age2 = Console.ReadLine();
+
+            Console.Write("Height (in cm): ");
+            var height2 = float.Parse(Console.ReadLine()); ;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4842747)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4842747&bid=16077284)

**Pull request closed without merging**